examples/hacker_news_assets/hacker_news_assets/sensors/hn_tables_updated_sensor.py

```diff
@@ -11,13 +11,13 @@
 )
 
 
-def make_hn_tables_updated_sensor(job: JobDefinition) -> SensorDefinition:
+def make_hn_tables_updated_sensor(job_name: str) -> SensorDefinition:
```

Comment:
We don't need this `make` function anymore, right?

Reply:
We actually do - we use it in a bunch of places, so probably less boilerplate to keep the factory